product-matrix.json

```diff
@@ -60,6 +60,18 @@
       "dartPluginVersion": "221.5177",
       "sinceBuild": "221.5080.210",
       "untilBuild": "221.*"
+    },
+    {
+      "channel": "dev",
+      "comments": "IntelliJ 2022.2",
+      "name": "2022.2",
+      "version": "2022.2",
+      "ideaProduct": "ideaIC",
+      "ideaVersion": "222.2889.14",
+      "baseVersion": "LATEST-EAP-SNAPSHOT",
```

Comment:
Is this base version going to change over time? Not sure if it matters.

Reply:
It will become something different when the EAP version is released as stable.